### PR TITLE
Add x86_64 module to uclibc/mod.rs

### PR DESCRIPTION
**Pull request by humenda**

### CODE REVIEW

File: src/unix/uclibc/mod.rs [hunk 1]
```diff
@@ -1759,6 +1759,9 @@ cfg_if! {
     if #[cfg(target_arch = "mips")] {
         mod mips;
         pub use self::mips::*;
+    } if #[cfg(target_arch = "x86_64")] {
```
Comment (alexcrichton) [inline src/unix/uclibc/mod.rs]: I think this may need to be `else if`?

File: src/unix/uclibc/mod.rs [hunk 2]
```diff
@@ -1759,6 +1759,9 @@ cfg_if! {
...
+        mod x86_64;
+        use self::x86_64::*;
```
Comment (alexcrichton) [inline src/unix/uclibc/mod.rs]: This'll probably need to be `pub use`

**Comment by humenda** [comment]
Thanks for the watchful review! Issues are corrected.

**Pull request merged**